lib/Map/TableColumn.js

```diff
@@ -8,9 +8,9 @@ var destroyObject = require('terriajs-cesium/Source/Core/destroyObject');
 var DeveloperError = require('terriajs-cesium/Source/Core/DeveloperError');
 var JulianDate = require('terriajs-cesium/Source/Core/JulianDate');
 var knockout = require('terriajs-cesium/Source/ThirdParty/knockout');
+var uniq = require('lodash').uniq;
```

Comment:
I think this should be `var uniq = require('lodash.uniq');` cause otherwise webpack will build all of lodash into our app (no tree shaking here, unfortunately).